jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -40,8 +40,11 @@
      (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
    (let ((use-torso t))
      (if (and (member :use-torso args) (null (cadr (member :use-torso args)))) (setq use-torso nil))
-     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t)
+     (send* self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t args)
    ))
+  (:rarm-angle-vector 
+   (av &optional (tm 3000) &rest args)
+   (send* self :angle-vector av tm :ctype :arm-controller args))
```

Comment:
Passing ctype keyward in angle-vector solve the problem, and i think you should define which controller you want to use every time when you use angle-vector.

Anyway, why does head move?
When you solve ik with an arm, does head move?

BTW,
JSK default: you can set controller from angle-vector as below.
`(send *ri* : angle-vector av 3000 :default-controller 0)`

`:angle-vector` method should be same as pthe robot, and the problem might be `:angle-vector-motion-plan` only accepts keywards.
I worked on this issue on batereus, so if you want to know more, please take a look.